fastlane/Fastfile

```diff
@@ -22,6 +29,8 @@ CDN_URL = 'https://cdn.a8c-ci.services'
 # Use this instead of getting values from ENV directly
 # It will throw an error if the requested value is missing
 def get_required_env(key)
+  return 'dry-run' if DRY_RUN && (key == 'BUILDKITE_BUILD_NUMBER' || key == 'BUILDKITE_TAG' || key == 'SLACK_WEBHOOK')
```

Comment:
What is the use case for this change? I ask because I'm afraid it adds some surprising behavior. I might be overthinking it, but overriding those env var values when in dry run seems a bit strong, what if in dry run the code needs to read the actual value?